test/contracts/apps/disputable/disputable_app.js

```diff
@@ -142,8 +143,12 @@ contract('DisputableApp', ([_, owner, agreement, anotherAgreement, someone]) =>
         await disputable.setAgreement(agreement.address, { from: owner })
       })
 
-      it('does not revert', async () => {
-        await disputable.newAction(0, '0x00', owner)
+      it('submits a new action', async () => {
+        const { receipt } = await disputable.newAction(0, '0x00', owner)
+
+        const logs = decodeEventsOfType(receipt, AgreementMock.abi, 'NewAction')
+        assertAmountOfEvents({ logs }, 'NewAction')
+        assertEvent({ logs }, 'NewAction', { disputableActionId: 0, context: '0x00', submitter: owner })
```

Comment:
Minor nitpick, but we could use embed use the `decodeAbiFor` option now :)

Reply:
We don't have the helpers here yet haha